content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,162 @@
+
+   ![](/img/ws-image10.png)
+2. Type in the domain you want to enable SSO on:
+
+   ![](/img/glp_domain.jpg)
+3. Input the metadata from the step above.
+
+   While HPE GreenLake edge-to-cloud platform does support entering this information manually, it's recommended that you simply upload the XML metadata that was downloaded in the previous step. To do so, Select Metadata File, selecting the XML file. Then, click Next.
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta. Set the idle timeout value as well.
+
+   ![](/img/config_setting_sso_appjpg.jpg)
+5. Then click **Next**.    
+6. Create a recovery user so that, in the event SSO fails, an admin will still be able to access the HPE GreenLake edge-to-cloud platform.    
+
+   ![](/img/recovery_user.jpg)
+
+   Congratulations! SSO will now be enabled for HPE GreenLake edge-to-cloud platform as well as the HPE GreenLake Private Cloud Enterprise application. Log out and on the HPE GreenLake edge-to-cloud platform home page, click **Sign in with SSO**.
+
+**Testing and troubleshooting:**
+
+On the HPE GreenLake edge-to-cloud platform home page, click **Sign In with SSO**.
+
+![](/img/ws-image15.png)
+
+![](/img/hpe-greenlake-sso-page.jpg)
+
+Enter the SSO credentials. You will be redirected to Okta to authenticate. Once you successfully authenticate, you will be redirected back to HPE GreenLake edge-to-cloud platform. You can then click on the HPE GreenLake Private Cloud Enterprise application and be given access based on the configured role/permissions.
+
+**Additional notes:**
+
+* There must be at least **one** verified user belonging to the **Domain** prior to configuration.    
+* In order to configure SSO, you must be logged into the HPE GreenLake edge-to-cloud platform with a user from the domain.
+* SSO user access is determined by the “role_name” attribute included in the SAML hpe_ccs_attribute provided by the IdP.    
+* For more troubleshooting: <https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us>
+* Customer users should be given access to SAML application.
+* After authentication when clicking the HPE GreenLake edge-to-cloud platform for Private Cloud Enterprise application**,** if it leads to the below error, it will take 1 hr to sync. If it does not do so within that time period, the customer should contact their HPE administrator. 
+
+I hope this blog post answers any questions you may have had in regards to how to configure single sign-on for HPE GreenLake Private Cloud Enterprise on the HPE GreenLake edge-to-cloud platform using Okta Active Directory. Please return back to the [HPE Developer blog](https://developer.hpe.com/blog) for more tips and tricks on working with the HPE GreenLake edge-to-cloud platform. If you have any further questions, please feel free to reach out to us via the #glc-support Slack channel.
```

Comment:
You may want to hotlink the "#glc_support" slack channel.

BTW, is the slack channel publicly available or is it only on the HPE Internal slack workspace? If it is only for HPE Internal people, I would not advise to mention the slack channel in the conclusion of the blog post. The blog post will be made available to internal and external community members.